infra/aws/terraform/prow-build-cluster/vpc.tf

```diff
@@ -65,6 +65,7 @@ module "vpc" {
   public_subnet_tags = {
     "kubernetes.io/role/elb"                    = 1
     "kubernetes.io/cluster/${var.cluster_name}" = "owned"
+    "karpenter.sh/discovery"                    = var.cluster_name
```

Comment:
I think we want to put this in the private subnets, no? do we have public facing instances?

Reply:
I don't think so, but I could be mistaken. We want our instances to have public IP addresses, the only way we managed to do this is by putting them in public subnets. This is required because jobs were running into rate limits (e.g. from GitHub) when behind NAT, so we had to rule it out.

Reply:
ah ok - yes I think I remember seeing/hearing that